src/components/CreateBeer.js

```diff
@@ -0,0 +1,57 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+function CreateBeer() {
+
+    const [name, setName] = useState("");
+    const [tagline, setTagline] = useState("");
+    const [description, setDescription] = useState("");
+    const [first_brewed, setFirstBrewed] = useState("");
+    const [brewers_tips, setBrewersTips] = useState("");
+    const [attenuation_level, setAttenuation] = useState();
+    const [contributed_by, setContributed] = useState("");
+    const [newBeer, setNewBeer] = useState({});
+    let navigate = useNavigate();
+
+    useEffect(() => {
```

Comment:
You don't need useEffect when creating items. You're not fetching any data, but only send it one time when you click submit button. 

So you want to have axios post request inside a handleForm method